dspace/config/modules/rest.cfg

```diff
@@ -39,11 +39,6 @@ rest.report-url.collections = static/reports/index.html
 rest.report-url.item-query = static/reports/query.html
 #rest.report-url.custom = 
 
-##### database specific way to format a regex SQL clause #####
-# The REST Report Tools may pass a regular expression test to the database.  
-# The following configuration setting will construct a SQL regular expression test appropriate to your database engine
-rest.regex-clause = text_value ~ ?
```

Comment:
This seems to be an accidental change.  Why is this configuration being removed in this PR?

Reply:
JPA2 does not support SQL restrictions: https://stackoverflow.com/questions/5182701/equivalent-of-hibernates-restrictions-sqlrestriction-in-jpa2-criteria-api

DSpace 6 is using one instance of this property to support a regex check in a query: https://github.com/DSpace/DSpace/blob/dspace-6.3/dspace-api/src/main/java/org/dspace/content/dao/impl/ItemDAOImpl.java#L188

@terrywbrady is it a problem that we remove regex restriction support in the old rest api?

Reply:
@Raf-atmire , this feature was added to allow non-postgres instances to use the rest report tools.

Are you also removing the code where this property is used?  If so, the rest report tools will break as a part of these changes.